src/scrapers/tv_show/ShowMerger.cpp

```diff
@@ -224,9 +224,13 @@ void copyDetailsToShow(TvShow& target, TvShow& source, const QSet<ShowScraperInf
 
 void copyDetailsToEpisode(TvShowEpisode& target, const TvShowEpisode& source, const QSet<EpisodeScraperInfo>& details)
 {
+    bool hasThumbnail = false;
     for (EpisodeScraperInfo detail : details) {
+        if (detail == EpisodeScraperInfo::Thumbnail)
+            hasThumbnail = true;
         copyDetailToEpisode(target, source, detail);
     }
+  target.setWantThumbnailDownload(hasThumbnail);
```

Comment:
```suggestion
    target.setWantThumbnailDownload(hasThumbnail);
```

Reply:
Btw. You could avoid the check in the loop by using `details.contains(EpisodeScraperInfo::Thumbnail)` :-)